### ISSUE TITLE
undefined symbol: _ZN5Graph8plotBandER9EigParserR12ConfigParserPS_NS_9GraphSaveE

### ISSUE THREAD

**Issue by PabloPerez-Martin** [piti-diablotin/qAgate#2]
Hello,
for a while now, I have been using qAgate to look at the results of some MD simulations. Suddenly, I have become unable to use it, receiving the error message: 
qAgate: symbol lookup error: qAgate: undefined symbol:_ZN5Graph8plotBandER9EigParserR12ConfigParserPS_NS_9GraphSaveE
whenever I run it on the terminal.

I do not know if this could actually be a problem in my side, I have been updating libraries and this could have triggered the problem. I have, however, tried a fresh installation of qAgate on a new machine and still have encountered the same problem. Is this a known issue?

I would also like to take the chance to thank you for developing this wonderful tool. It definitely has made a breeze of looking into the results of the simulations.

Regards,
Pablo

**Comment by ELILAME** [comment]
Yeah, I have the same problem. When I install the qAgate from PPA contains package in Ubuntu, the installation was successful but I could not use it with the error message:
qAgate: symbol lookup error: qAgate: undefined symbol:_ZN5Graph8plotBandER9EigParserR12ConfigParserPS_NS_9GraphSaveE

So, I tried to compile qAgate from sources and  got an error message:
qdispersion/qdispersion.cpp:249:60: error: no matching function for call to ‘Graph::plotBand(EigParser&, ConfigParser&, Gnuplot*, Graph::GraphSave&)’

I could not found the definition of this function and don't know the useage of it. I just choose to delete it in: qAgate/qdispersion/qdispersion.cpp  ##delete 241-257
qAgate/qtdep/qtdep.cpp  ##delete 114
Then, conutining to the compile qAgate, and as a result it could work. Thank you for your work in developing a great tool !!  if you have time , please take our quetions into account.
